launchdarkly-server-sdk/src/client.rs

```diff
@@ -1393,6 +1415,30 @@ mod tests {
         assert_eq!(event_rx.iter().count(), 0);
     }
 
+    #[test]
```

Comment:
I think it would be nice to have a daemon mode test that initialized from a store and evaluated a flag that did exist, and there were no undesirable logs.